cards_main.py

```diff
@@ -194,6 +213,9 @@ def playTrick(self, start):
 		if self.trickNum == 0:
 			startPlayer = self.players[start]
 			addCard = startPlayer.play(option="play", c='2c')
+			print(type(addCard))
```

Comment:
```suggestion
			print(type(addCard))
```
What's the purpose of this line